bucket/networkmanager.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "2021.1.0",
+    "description": "Tool for managing networks and network troubleshooting",
+    "homepage": "https://borntoberoot.net/NETworkManager/",
+    "license": "GPL-3.0-only",
+    "suggest": {
+        ".NET": "dotnet-sdk"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/BornToBeRoot/NETworkManager/releases/download/2021.1.0/NETworkManager_2021.1.0_Archiv.zip",
+            "hash": "425517a996ce52bb2adff05d02537817c954329402aae9f05773b36c61997e71"
+        }
+    },
+    "bin": "NETworkManager.exe",
+    "extract_dir": "NETworkManager",
+    "shortcuts": [
+        [
+            "NETworkManager"
```

Comment:
What??

Reply:
By binary, I understood  `*.exe`. So guessing by you reaction, you meant to remove shortcut without `.exe` suffix?
I am long term linux user and now I am forced to use to use Windows, but catching up is not easy...

Reply:
Now I get it, you meant that "bin" line.